packages/sanity/src/core/search/common/utils.ts

```diff
@@ -5,7 +5,8 @@ import {isNonNullable} from '../../util/isNonNullable'
 const isDocumentType = (type: SchemaType): type is ObjectSchemaType =>
   Boolean(type.type && type.type.name === 'document')
 
-const isSanityType = (type: SchemaType): boolean => type.name.startsWith('sanity.')
+const isSanityType = (type: SchemaType): boolean =>
```

Comment:
What do you think about renaming this function to something like `isExcludedType` or `isIgnoredType`? It'd more explicitly match what the function does now.

Reply:
Sure, that makes sense to me